### ISSUE TITLE
Qobuz Support - Lidarr - Add Qobuz support to the script

### ISSUE THREAD

**Issue by Simpuhl** [RandomNinjaAtk/arr-scripts#77]
**Is your feature request related to a problem? Please describe.**
Add Qobuz (https://www.qobuz.com) support, which similar to Deezer and tidal

**Describe the solution you'd like**
It would be nice to have a third option for downloading flacs.

**Describe alternatives you've considered**
None

**Additional context**
I know that there are "bot" discord channels that are able to download from Qobuz so I thought it might be cool to add it to the arr script. They even offer 3 month free subscription.

**Comment by RandomNinjaAtk** [comment]
I don't believe there is a viable method to make this work and the 3 included solutions should be enough.... I will gladly review a PR for it, if someone takes the time implement it...

**Comment by Simpuhl** [comment]
I am not a huge coder but if anybody does want to take this on, theres a few tools here on github that do this and maybe check out the way they do it and impliment it here. 

If I understand correctly, all your scripts are written in Bash?

Examples: 

1. https://github.com/vitiko98/qobuz-dl
2. https://github.com/nathom/streamrip

**Comment by RandomNinjaAtk** [comment]
The problem isn't the client, its the process to search/match to tell the client what to download...

**Comment by Simpuhl** [comment]
I found this container, this guy figured out how to do the search but this is a Windows Tool, not docker. 

https://github.com/DJDoubleD/QobuzDownloaderX-MOD

In case anybody is interested

**Comment by Pheggas** [comment]
I would kill for this feature

**Comment by yodatak** [comment]
You could maybe use https://github.com/nathom/streamrip
that use python and as a library for it ( it works great and support  A scriptable music downloader for Qobuz, Tidal, SoundCloud, and Deezer )

**Comment by RandomNinjaAtk** [comment]
@yodatak 

Did you even read this thread…?

The issue has never been the client… PR welcome…

**Comment by knguyen1** [comment]
Jackett would have to recognize Qobuz as an indexer.
